server/channels/api4/ldap.go

```diff
@@ -130,31 +129,15 @@ func testLdapFilters(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	var settings model.LdapSettings
-	if err := json.NewDecoder(r.Body).Decode(&settings); err != nil {
-		c.SetInvalidParamWithErr("ldap_settings", err)
-		return
-	}
-
-	res, appErr := c.App.TestLdapFilters(c.AppContext, &settings)
-	if appErr != nil {
-		c.Err = appErr
+	testTypeStr := r.URL.Query().Get("test")
+	if testTypeStr == "" {
+		c.SetInvalidParam("test")
```

Comment:
@cpoile no security concern, but why a query parameter instead of a request body parameter?